code/modules/reagents/reagents/toxin.dm

```diff
@@ -486,10 +485,12 @@
 			power = (2*effect_str) //While stamina loss is going, stamina regen apparently doesn't happen, so I can keep this smaller.
 			L.reagent_pain_modifier -= PAIN_REDUCTION_LIGHT
 		if(21 to 45)
+			purge_rate = (purge_rate == 1) ? 1 : 0.2
 			power = (6*effect_str)
 			L.reagent_pain_modifier -= PAIN_REDUCTION_HEAVY
 			L.jitter(4) //Shows that things are bad
 		if(46 to INFINITY)
+			purge_rate = (purge_rate == 1) ? 1 : 1
```

Comment:
just directly set the purge rate as 1 here, and 0.2 at l 488.
These check are useless